OnlineStore/OnlineStore.BusinessLogic/Services/JwtService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Configuration;
+using Microsoft.IdentityModel.Tokens;
+using OnlineStore.BusinessLogic.Interfaces;
+
+namespace OnlineStore.BusinessLogic.Services
+{
+    internal class JwtService : IJwtService
+    {
+        private readonly IConfiguration _configuration;
+
+        public JwtService(IConfiguration configuration)
+        {
+            _configuration = configuration ?? throw new ArgumentNullException(nameof(configuration));
+        }
+        
+
+        public async Task<String> GetJwtTokenString(List<Claim> claims)
+        {
+            claims.Add(new Claim(JwtRegisteredClaimNames.Jti, Guid.NewGuid().ToString("D")));
+            claims.Add(new Claim(JwtRegisteredClaimNames.Iat, DateTime.UtcNow.ToString("R")));
+
+            var SecurityKey = _configuration["Jwt:SecurityKey"];
```

Comment:
It's not mandatory at this stage, but it's better not to pass configuration explicitly and use IOptions instead